templates/shortcodes/sdk_table.html

```diff
@@ -0,0 +1,30 @@
+{% set sdks = load_data(path="content/ecosystem/sdks/sdks.toml") %}
+<table>
+    <thead>
+        <tr>
+            <th>Project</th>
+            <th>Language</th>
+            <th>Licence</th>
+            <th>Repository</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for name, sdk in sdks %}
+    {% if not featured and sdk.featured_in is not containing(type) %}
+        {% set display = true %}
+    {% elif featured and sdk.featured_in is containing(type) %}
+        {% set display = true %}
```

Comment:
```suggestion
    {% if featured and sdk.featured_in is containing(type)
        or not featured and not sdk_featured_in
        or not featured and sdk_featured_in is not containing(type) %}
        {% set display = true %}
```

Reply:
(I checked, while Tera doesn't have parentheses, `and` has higher precedence than `or`, like everywhere else)

Reply:
Is it safe to assume that `or not featured and not sdk_featured_in` is for the case where there is no `featured_in` for a specific sdk in `sdks.toml`?

I guess `containing(type)` returns false if `featured_in` does not exist, and `not containing(type)` returns true… which seems to already cover the case?

Is `sdk_featured_in` a typo for `sdk.featured_in`?

Reply:
Yes, that one underscore should be a period, and I was assuming that `containing(type)` would throw an error if the LHS is undefined but if it just returns `false`, you can omit the middle condition.

Reply:
It does throw an error… but it still does with the suggestion (after s/_/./)

It looks like it's trying to evaluation the if as a whole and doesn't stop at the first `or`. I'm tempted to make the `featured_in` field mandatory in the toml to avoid over-complexifying the template. Wdyt?

Reply:
So it doesn't short-circuit? 😬
I would go for nested ifs then, but that's just personal preference. We can keep the field mandatory.